.travis.yml

```diff
@@ -16,12 +19,14 @@ before_install:
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
 
 before_script:
-  - npm install -g jpm mozilla-download
+  - npm install -g jpm
```

Comment:
Sure, for `cp`, but I'd still argue that this:

```sh
$ time cp *.png tmp/
# 0.00s user 0.01s system 92% cpu 0.013 total
```

Is going to be marginally faster than copying files one at a time, like this:

```sh
$ time (cp alpha-badge.png tmp2 && cp bgtexture.png tmp2 && cp chrome-store-icon.png tmp2 && cp clipboard-16.png tmp2 && cp clipboard-32.png tmp2 && cp clock.png tmp2 && cp close.png tmp2 && cp comment-bubble-open.png tmp2 && cp comment-bubble.png tmp2 && cp comment.png tmp2 && cp copy-image-16.png tmp2 && cp copy-image-32.png tmp2 && cp down-arrow.png tmp2 && cp email-16.png tmp2 && cp email-32.png tmp2 && cp facebook-16.png tmp2 && cp facebook-32.png tmp2 && cp flag.png tmp2 && cp garbage-bin.png tmp2 && cp help-recall.png tmp2 && cp help-shooter.png tmp2 && cp icon-recall.png tmp2 && cp icon-shoot-active.png tmp2 && cp icon-shoot.png tmp2 && cp link-16.png tmp2 && cp link-32.png tmp2 && cp menu.png tmp2 && cp my-shots.png tmp2 && cp pageshot-icon-128.png tmp2 && cp pageshot-icon-16.png tmp2 && cp pageshot-icon-19.png tmp2 && cp pageshot-icon-256.png tmp2 && cp pageshot-icon-32.png tmp2 && cp pageshot-icon-38.png tmp2 && cp pageshot-icon-48.png tmp2 && cp pageshot-invert-16px.png tmp2 && cp pageshot-invert-32px.png tmp2 && cp pageshot-invert-64px.png tmp2 && cp pageshot-mac-16px.png tmp2 && cp pageshot-mac-32px.png tmp2 && cp pageshot-mac-64px.png tmp2 && cp pageshot-win-16px.png tmp2 && cp pageshot-win-32px.png tmp2 && cp pageshot-win-64px.png tmp2 && cp pinterest-16.png tmp2 && cp pinterest-32.png tmp2 && cp profile-open.png tmp2 && cp profile.png tmp2 && cp twitter-16.png tmp2 && cp twitter-32.png tmp2 && cp up-arrow.png tmp2)
# 0.04s user 0.08s system 98% cpu 0.121 total
```

But yeah, point taken that the real bottleneck is browserify and tackling that would have the best performance boost.